docs/docs/integrations/platforms/index.mdx

```diff
@@ -38,6 +38,7 @@ These providers have standalone `langchain-{provider}` packages for improved ver
 - [OpenAI](/docs/integrations/platforms/openai)
 - [Pinecone](/docs/integrations/providers/pinecone)
 - [Qdrant](/docs/integrations/providers/qdrant)
+- [Qdrant - NEW](/docs/integrations/providers/qdrant-new)
```

Comment:
broken link? qdrant-new was a vectorstore not a provider. also please merge into one entry

Reply:
[`67cb5aa` (#24166)](https://github.com/langchain-ai/langchain/pull/24166/commits/67cb5aad6ad70ae1eb44aa4059b68633eee83abf)

Reply:
I had added this entry before creating the page. I incorrectly predicted the location.
Thanks.